### PR TITLE
Setup Project

### PR DESCRIPTION
**Pull request by wwwdrt**
<h1 align="center">Hey CRs</h1>

<div align="center">
Please feel free to reach out to me on Slack anytime. You can send me a direct message or give me a call. If you choose to call, please introduce yourself so I know who I'm speaking with. I'm here to help, so don't hesitate to get in touch 😁
</div>

<br>

<div align="center">
  

### Latest project 🔥"[AI Quiz Generation Platform](https://github.com/wwwdrt/quizzen)"
</div>

<h1 align="center">Changes</h1>

<div align="center">
In this branch, I have configured the base project.
</div>

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const BookList = () => <div>BookList</div>;
```
Comment (Munsa1) [inline src/components/BookList.js:1]: - [ ] As per requirement, your application should have 2-page components, one page should contain the following:

```
A <Books> component, which contains the following components:


<BookList>

Renders a list of books using the <Book> component



<Book>

Takes the book data as a prop and renders the title and the author
Has a button for removing books

the button wouldn't remove the books yet, this is just a placeholder for now





<NewBookForm>

Renders a form with inputs for the title and the author of the book
Has a button to submit

the button wouldn't add the book yet, this is just a placeholder for now

```
The other component should be the categories page

Kindly refactor your codebase to meet these requirements

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @wwwdrt
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly remove the link to the `Live Demo` section from your table of contents for a more descriptive documentation
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @wwwdrt, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Made Some Requested Changes :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const Book = () => (
+  <div>
+    <h2>Book Title</h2>
+    <p>Author</p>
+    <button type="button">Delete</button>
+  </div>
+);
```
[the remark this answers is not in the document]
Comment (peterrobert) [reply src/components/Book.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @wwwdrt , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**